readme.md

```diff
@@ -82,13 +86,32 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ output }) => {
```

Comment:
Why `output` instead of `files`?

Reply:
No reason at all, shall we switch to `files`?

Reply:
Yes, please do 👍

Reply:
Updated in the latest commit.